Assignments/stu3/api.js

```diff
@@ -38,17 +38,21 @@ app.get('/ex2/transaction-types/', cors(corsOptions), async (req, res) => {
 });
 
 
-//
 // EX 3
-//
 
 app.get('/ex3/users/', cors(corsOptions), async (req, res) => { 
     let year = req.query ['dobFilterYear']
     let users = await dataAccess.getUsersByDob(1972)
     res.send(users)
-    //res.send('test')
-});
+})
+
+// EX 4
 
+app.get('/ex4/users/:id/accounts', cors(corsOptions), async (req, res) => { 
+    let userId = req.query ['id']
+    let accounts = await dataAccess.getAccountsByUserId(1)
```

Comment:
We should remove the 1 here because we can't hardcode this value.  Replace the 1 with `userId` and you should be good to go.

Reply:
Where should I put the 1 if the bank user id I'm requesting information on is 1?

Reply:
Understood. Thank you!